### PR TITLE
C#: Add VSCode `launch.json`

### PR DESCRIPTION
**Pull request by tamasvajk**
The PR is adding various VSCode debug launch configurations for C# extraction.

### CODE REVIEW

File: csharp/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            "program": "${workspaceFolder}/extractor/Semmle.Extraction.CSharp.Driver/bin/Debug/net8.0/Semmle.Extraction.CSharp.Driver.dll",
+            "stopAtEntry": true,
+            "args": [
+                "--binlog",
+                "${workspaceFolder}/ql/integration-tests/all-platforms/binlog/test.binlog"
+            ],
+            "env": {}
+        },
+    ]
+}
```
Comment (michaelnebel) [inline csharp/.vscode/launch.json:65]: Should we also have an entry for "attach"?
```
{
            "name": ".NET Core Attach",
            "type": "coreclr",
            "request": "attach",
            "processId": "${command:pickProcess}",
            "requireExactSource": false
        },
```

Comment (tamasvajk) [reply csharp/.vscode/launch.json:65]: I don't mind adding it, but I haven't used that lately. Also, attaching to the running extractor process likely requires some changes to your local sembuild commands too, to build a bundle with debug artifacts.

Comment (michaelnebel) [reply csharp/.vscode/launch.json:65]: Yes. It also looks like this code has changed a lot recently, so I am not sure how to change the build to produce debug artifacts. Maybe we just need to change Release -> Debug in `ql/csharp/scripts/create-extractor-pack.sh`.
So this is out of scope for this PR.

**Review by michaelnebel — APPROVED** [review]
LGTM

**Pull request merged**